bin/nic.pl

```diff
@@ -313,16 +313,23 @@ sub getHomeDir {
 }
 
 sub loadConfig {
-	open(my $cfh, "<", getHomeDir()."/.nicrc") or return;
-	while(<$cfh>) {
-		if(/^(.+?)\s*=\s*\"(.*)\"$/) {
-			my $key = $1;
-			my $value = $2;
-			$CONFIG{$key} = $value;
-		}
-	}
+    open(my $cfh, "<", getHomeDir()."/.nicrc") or return;
+    while(<$cfh>) {
+        # Remove leading and trailing whitespace
+        s/^\s+|\s+$//g;
```

Comment:
We can avoid having to do this by also matching anything with a leading space, as you handle the leading/trailing below
```perl
/^\s*(.+?)\s*=\s*"(.*)"$/
```